claim_ai_quality/communication_interface/aiServerCommunicationInterface.py

```diff
@@ -59,7 +59,8 @@ async def _get_async_data(self):
     @database_sync_to_async
     def _get_imis_data(self):
         queryset = Claim.objects\
-            .filter(json_ext__jsoncontains={'claim_ai_quality': {'was_categorized': False}})\
+            .filter(json_ext__jsoncontains={'claim_ai_quality': {'was_categorized': False}},
+                    validity_to__isnull=True)\
```

Comment:
We should add here 'claim_ai_quality': {'was_categorized': False, 'request_time': null}. To make sure we don't send twice a Claim.

Reply:
Filter added